domains/Contact/ManageContact/Web/Controllers/ContactController.php

```diff
@@ -77,11 +77,13 @@ public function store(Request $request, int $vaultId)
     public function show(Request $request, int $vaultId, int $contactId)
     {
         $vault = Vault::findOrFail($vaultId);
-        $contact = Contact::with('gender')
-            ->with('pronoun')
-            ->with('notes')
-            ->with('dates')
-            ->with('vault')
+        $contact = Contact::with([
+            'gender',
+            'pronoun',
+            'notes',
+            'importantDates',
+            'vault',
+        ])
```

Comment:
Is this purely cosmetic, or does this change performance?

Reply:
No, but I've also renamed the `dates`